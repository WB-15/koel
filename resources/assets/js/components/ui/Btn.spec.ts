import { expect, it } from 'vitest'
import ComponentTestCase from '@/__tests__/ComponentTestCase'
import Btn from './Btn.vue'

new class extends ComponentTestCase {
  protected test () {
    it('renders', () => {
      expect(this.render(Btn, {
        slots: {
          default: 'Click Me Nao'
        }
      }).html()).toMatchSnapshot()
    })
  }
}
