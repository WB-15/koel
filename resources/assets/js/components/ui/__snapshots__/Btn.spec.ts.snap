// Vitest Snapshot v1

exports[`renders 1`] = `<button type="button" data-v-27deb898="">Click Me Nao</button>`;
